package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "ipfs-repo-migrations",
+  "version": "0.1.0",
+  "description": "Migration framework for versioning of JS IPFS Repo",
+  "keywords": [
+    "IPFS",
+    "libp2p",
+    "migrations"
+  ],
+  "homepage": "https://github.com/ipfs/js-ipfs-repo-migrations",
+  "bugs": {
+    "url": "https://github.com/ipfs/js-ipfs-repo-migrations/issues/"
+  },
+  "license": "MIT",
+  "leadMaintainer": "Adam Uhlir <adam@uhlir.dev>",
```

Comment:
FYI we use `maintainer.json` for this info now e.g. https://github.com/ipfs/js-ipfs/blob/master/maintainer.json